spot-client/src/common/remote-control/screenshare-connection.js

```diff
@@ -116,7 +129,7 @@ export default class ScreenshareConnection {
                      * Assume the connection was lost if LOCAL_TRACK_STOPPED
                      * fires but stop() was not explicitly called.
                      */
-                    if (this._isActive) {
+                    if (this._isActive || deferredStart) {
```

Comment:
Just what the comment says, it will call stop but only if it has not been called already. In the deferredStart scenario it's never active when it stops. Hm so it actually means this will not work as originally intended for the deferredStart, because it always ignores the active flag